frontend/src/pages/accounts/[id].jsx

```diff
@@ -4,7 +4,7 @@ import React from "react";
 
 import TransactionIcon from "../../../public/static/images/icon-t-transactions.svg";
 
-import AccountsApi from "../../libraries/explorer-wamp/accounts";
+import AccountApi from "../../libraries/explorer-wamp/accounts";
```

Comment:
Let's rename it to `AccountsApi` in the `libraries/explorer-wamp/accounts` to be consistent with the existing *Api naming.

Reply:
got it